src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Well, there are some cases where it can't exactly identify the page, so it just picks the first available target. If the page is slow to load and doesn't show up in the results for a second, if there is no URL like if this is 'attach' with one tab open, or if the URL doesn't quite match. These aren't uncommon. Accidentally attaching to an extension is a confusing experience so I want to avoid it.